README.md

```diff
@@ -32,6 +32,8 @@ This library does not build software like rpmbuild. It is meant for finished art
 ### Examples
 
 ```rust
+use std::str::FromStr;
```

Comment:
Isn't this part of the prelude?

Reply:
I don't think so.  In any case I was getting some compile errors without it when I added an example.

I removed the example because we already have one in `lib.rs` and having the same code in 3 places is excessive.  But it's there too.